README.md

```diff
@@ -14,6 +14,17 @@ This installs the module from npm and adds it as a dependency to the application
 
 If you create a MongoDB data source using the data source generator as described below, you don't have to do this, since the generator will run `npm install` for you.
 
+## Supported versions
+
+**Starting from the version 6.0.0, this connector is no longer compatible with LoopBack 3. Please use the latest 5.x version in your LoopBack 3 applications.**
+
+This module adopts the [Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy, with the following End Of Life (EOL) dates:
+
+| Version    | Status          | Published | EOL                  | LoopBack | Juggler  |
+| ---------- | --------------- | --------- | -------------------- | ---------|----------|
+| 6.x        | Current         | Nov 2020  | Apr 2023 _(minimum)_ | 4        | 4.x      |
+| 5.x        | Active LTS      | Jun 2019  | Aug 2021             | 3, 4     | 3.x, 4.x |
```

Comment:
Much better! 👏🏻 

Let's add a row for the 4.x version of the connector please, it should be listed as "Maintenance LTS".

Reply:
What is the EOL of 4.x?

Reply:
We are following https://github.com/CloudNativeJS/ModuleLTS.

> When a version of Node.js enters LTS, whichever is the latest major version (X.y.z) of an LTS adopting module at that time will be supported for at least security and critical fixes for the lifetime of that version of Node.js.

loopback-connector-mongodb 5.x was released in Jun 2019, at which time the latest LTS version of Node.js was 10.x.  Before our release, people were using loopback-connector-mongodb 4.x with Node.js 10.x, thus we need to support connector 4.x for at least as long as Node.js will support 10.x - that means 2021-04-30 - see https://github.com/nodejs/Release

As for loopback-connector-mongodb 5.x, we need to support it until the current Node.js LTS version 14.x goes end of life, which means April 2023. Please update the row in the table accordingly.